tests/data/aws/dynamodb.py

```diff
@@ -0,0 +1,254 @@
+                    },
+                    {
+                        "AttributeName": "sample_2",
+                        "AttributeType": "B"
+                    },
+                    {
+                        "AttributeName": "sample_3",
+                        "AttributeType": "C"
+                    }
+                ],
+                "GlobalSecondaryIndexes": [
+                    {
+                        "IndexSizeBytes": 11111111,
+                        "IndexName": "sample_index_1",
+                        "Projection": {
+                            "ProjectionType": "ALL"
+                        },
+                        "ProvisionedThroughput": {
+                            "WriteCapacityUnits": 30,
+                            "ReadCapacityUnits": 30
+                        },
+                        "IndexStatus": "ACTIVE",
+                        "KeySchema": [
+                            {
+                                "KeyType": "HASH",
+                                "AttributeName": "sample_schema"
+                            }
+                        ],
+                        "IndexArn": "arn:aws:dynamodb::table/example-table/index/sample_1-index",
+                        "ItemCount": 1000000
+                    },
+                    {
+                        "IndexSizeBytes": 22222222,
+                        "IndexName": "sample_index_2",
+                        "Projection": {
+                            "ProjectionType": "ALL"
+                        },
+                        "ProvisionedThroughput": {
+                            "WriteCapacityUnits": 40,
+                            "ReadCapacityUnits": 40
+                        },
+                        "IndexStatus": "ACTIVE",
+                        "KeySchema": [
+                            {
+                                "KeyType": "HASH",
+                                "AttributeName": "sample_schema"
+                            }
+                        ],
+                        "IndexArn": "arn:aws:dynamodb::table/example-table/index/sample_2-index",
+                        "ItemCount": 1000000
+                    }
+                ],
+                "ProvisionedThroughput": {
+                    "NumberOfDecreasesToday": 10,
+                    "WriteCapacityUnits": 10,
+                    "LastIncreaseDateTime": 1000000000.000,
+                    "ReadCapacityUnits": 10,
+                    "LastDecreaseDateTime": 1000000000.000
+                },
+                "TableSizeBytes": 100000000,
+                "TableName": "example-table",
+                "TableStatus": "ACTIVE",
+                "StreamSpecification": {
+                    "StreamViewType": "SAMPLE_STREAM_VIEW_TYPE",
+                    "StreamEnabled": True
+                },
+                "TableId": "00000000-0000-0000-0000-000000000000",
+                "LatestStreamLabel": "0000-00-00000:00:00.000",
```

Comment:
Does the API return this as a string or a datetime object?

Reply:
This should be a string from what I read in the boto3 documentation, but I changed some other parameters to datetime objects which I incorrectly made strings.

Reply:
I think Evan's pointing out that our versions of Neo4j (3.2 and 3.3) don't support Python datetime objects so you need to run `str()` any datetimes before loading them in to Neo4j.  It doesn't look like your `load_*` functions ingest any datetime objects as far as I can tell though.